microsoft/testsuites/dpdk/dpdkutil.py

```diff
@@ -220,6 +220,15 @@ def enable_uio_hv_generic_for_nic(node: Node, nic: NicInfo) -> None:
     echo = node.tools[Echo]
     lsmod = node.tools[Lsmod]
     modprobe = node.tools[Modprobe]
+    kernelcfg = node.tools[kernel_config.KernelConfig]
+
+    # check if kernel config for Hyper-V VMBus is enabled
+    config = "CONFIG_UIO_HV_GENERIC"
+    if not kernelcfg.is_enabled(config):
+        raise LisaException(
```

Comment:
use skipexception

Reply:
why?

Reply:
debian has older kernel version like 4.9, it doesn't contain this module, it is expected, maybe we can also add minimal kernel version check here.
```
prompt: Generic driver for Hyper-V VMBus
type: tristate
depends on: [CONFIG_HYPERV](https://cateee.net/lkddb/web-lkddb/HYPERV.html)
defined in [drivers/uio/Kconfig](https://github.com/torvalds/linux/tree/master/drivers/uio/Kconfig)
found in Linux kernels: 4.10–4.20, 5.0–5.19, 6.0, 6.1-rc+HEAD
modules built: uio_hv_generic
```

Reply:
I was wondering for the reason for "use skipexception"?

Reply:
I think checking for kconfig is more generic than a minimal kernel version check.

Reply:
skipexception means current behavior is expected, we just skip this testing on current distro. is this clear?